atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbMetricNames.java

```diff
@@ -96,4 +96,11 @@ private CellFilterMetrics() {
 
     public static final String LEGACY_READ = "legacyRead";
     public static final String LEGACY_WRITE = "legacyWrite";
+
+    public static final String CACHE_HITS = "lockWatchCacheHits";
+    public static final String CACHE_MISSES = "lockWatchCacheMisses";
+    public static final String CACHE_SIZE = "lockWatchCacheSize";
+    public static final String CACHE_RATIO_USED = "lockWatchCacheRatioUsed";
+    public static final String EVENT_CACHE_VALIDATION_FAILURES = "eventCacheFallbackCount";
+    public static final String VALUE_CACHE_VALIDATION_FAILURES = "valueCacheFallbackCount";
```

Comment:
I think the constant name should match the metric name (which based on later usage is what this metric actually is).